tests/unit_tests/layout/test_geo_coordinates.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+
+import warnings
+
+import pyproj
+import pytest
+
+from simtools.layout.geo_coordinates import GeoCoordinates
+from simtools.layout.telescope_position import TelescopePosition
+
+
+def LaPalma_reference_point():
```

Comment:
Why not use fixtures for these like we usually do?

Reply:
Conscious decision. Fixtures are making in this case simple code much more complicated.